ChineseFilter/sections/specific.txt

```diff
@@ -1096,6 +1096,7 @@ csdn.net###footerRightAds
 ||s3.amazonaws.com/dmas-public/sitemaji/vast.xml$domain=dailymotion.com
 ||c.mipcdn.com/static/*/mip-ad/mip-ad.js
 ||mycodes.net/js/*.js
+mcbbs.net###um + div
```

Comment:
Looks dangerours and unobvious. Please show screenshot of DOM inspector.

Reply:
Oh it's because the ID of the banner (and it's children elements) changes every time the page reloaded.

<img width="960" alt="20220819-00006" src="https://user-images.githubusercontent.com/4410086/185600763-23e7946b-3224-4f2f-a0d3-ea82ae9b4edf.png">
<img width="960" alt="20220819-00007" src="https://user-images.githubusercontent.com/4410086/185600770-a0fd2cbc-fb8f-43e3-9181-8597c650c155.png">

Reply:
Better upload saved page.